### PR TITLE
Variable header title and sub-title based on routing

### PR DESCRIPTION
**Pull request by ke4**
Changes:
- Changed title and sub title text
- Header title and sub-title depends on the page where the user is (Currently only the about page has different text)

### CODE REVIEW

**Review by jacobwindsor — APPROVED** [review]

**Review by MightyAx — DISMISSED** [review]

File: src/app/components/global-header/global-header.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
-<section class="vf-hero | vf-u-fullbleed vf-hero-hca" style="background-color: #3355aa">
+<section class="vf-hero vf-hero--800 | vf-u-fullbleed vf-hero-hca" style="background-color: #3355aa">
```
[the remark this answers is not in the document]
Comment (ke4) [reply src/app/components/global-header/global-header.component.html:1]: oh, good spot. We don't need that. That was my 1st try and I accidentally committed it. Sorry. I am going to remove that

File: src/app/components/global-header/global-header.component.css [hunk 2]
```diff
@@ -6,3 +6,7 @@
   padding: 2rem 0;
   padding: var(--vf-hero--spacing, 2rem) 0;
 }
+
+.vf-hero__content {
+  max-width: 80ch;
+}
```
[the remark this answers is not in the document]
Comment (ke4) [reply src/app/components/global-header/global-header.component.css:12]: this is a property change inside `vf-hero__content` class

**Review by MightyAx — DISMISSED** [review]
Glad my review was useful!

**Review by jacobwindsor — APPROVED** [review]

File: src/app/components/global-header/global-header.component.ts [hunk 3]
```diff
@@ -1,12 +1,40 @@
...
   ngOnInit(): void {}
+
+  activatedRoute = '';
+  showAboutLink = true;
+
+  get headerTitle() {
+      if (this.activatedRoute.includes('about')) {
+        return 'About the Catalogue';
+      } else {
+        return 'HCA Project Catalogue';
```
Comment (jacobwindsor) [inline src/app/components/global-header/global-header.component.ts:27]: You don't need the else here

File: src/app/components/global-header/global-header.component.ts [hunk 4]
```diff
@@ -1,12 +1,40 @@
...
+      }
+  }
+
+  get headerText() {
+    if (this.activatedRoute.includes('about')) {
+      this.showAboutLink = false;
+      return 'Aims, eligibility criteria and selection process';
+    } else {
+      this.showAboutLink = true;
+      return 'A comprehensive list of cellular resolution datasets for the Human Cell Atlas.';
```
Comment (jacobwindsor) [inline src/app/components/global-header/global-header.component.ts:37]: No else here either

**Review by jacobwindsor — CHANGES_REQUESTED** [review]

**Review by jacobwindsor — APPROVED** [review]

**Review by MightyAx — DISMISSED** [review]

**Review by jacobwindsor — APPROVED** [review]

**Review by MightyAx — APPROVED** [review]

**Pull request merged**